fahadmak/stackoverflow_clone
# 167137986 Filter Questions Functionality

**What does this PR do?**
This PR ensures that a user can filter questions by tags on the front page.
**Description of Task to be completed?**
- Create vote app
- create QuestionVote model
- create UpVote and DownVote View

**What are the relevant Pivotal Tracker stories?**
[#167137986](https://www.pivotaltracker.com/n/projects/2364284/stories/167137986)

**How should be tested manually?**
Run each of the following commands in your terminal in the respective order.

- `git clone https://github.com/fahadmak/stackoverflow_clone.git`
- `cd stackoverflow_clone`
- `git checkout ft-create-list-question-comments-167138181`
- `virtualenv venv`
- `. venv/bin/activate`
- `pip install -r requirements.txt`
- `run migrations using this command python manage.py migrate`
- `run python manage.py runserver 127.0.0.1:8000`
- `Go to this link in your browser 127.0.0.1:8000`
- `You will be redirected to the login page where you should log in`
- `Click on the 'Register Today' link to go to sign up page`
- `Try to sign up an account and then use it in the login page`
- Create a question or questions
- `Click on the tags below to see how they filter the questions`

**Screenshots**
<img width="1021" alt="Screen Shot 2019-07-23 at 12 28 51 PM" src="https://user-images.githubusercontent.com/13882936/61794353-b11ef900-ae29-11e9-85cd-fd0f1f032a20.png">
